cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help

```diff
@@ -29,14 +29,15 @@ OPTIONS
            and the service name.
 
        -o  <contract-file>
-           This option will specify the output location of the generated contract.
+           This option specify the output location of the generated contract.
 
        -s
```

Comment:
Is this still used? Shall we verify? @DimalChandrasiri

Reply:
+1, how ever OpenApiGenContractCmd does use it